contrib/ohi-release-notes/run.sh

```diff
@@ -3,13 +3,16 @@
 set -euo pipefail
 
 RT_PKG="github.com/newrelic/release-toolkit@latest"
+# TODO this is just added to check the CI tests are passing and should be reverted before merge , chicken egg thing.
+# DICTIONARY_URL="https://raw.githubusercontent.com/newrelic/release-toolkit/v1/contrib/ohi-release-notes/rt-dictionary.yml"
+DICTIONARY_URL="https://raw.githubusercontent.com/newrelic/newrelic-infra-checkers/main/rt-dictionary/.rt-dictionary.yml"
```

Comment:
Smart move here :)

Reply:
We may even revert it **after** merging (in a different PR) in order to avoid by-passing merge restrictions